README.rst

```diff
@@ -39,7 +39,35 @@ You will also need a deployed installation of edX lms and studio to run the test
 See `edx/configuration <http://github.com/edx/configuration>`_ for instructions on provisioning an edX instance.
 For additional information on how to setup your development environment, see `Developer Onboarding <https://openedx.atlassian.net/wiki/pages/viewpage.action?spaceKey=ENG&title=Developer+Onboarding#DeveloperOnboarding-Step4:Getreadytodevelop>`_
 
+OR Using Dockers
+------------
+
+1. Clone the repo:
+
+.. code:: bash
+    git clone https://github.com/edx/edx-e2e-tests
```

Comment:
The code snippets are not displaying in the rendered view.
I'm pretty sure that you need an empty line above them.